python/whylogs/api/writer/whylabs.py

```diff
@@ -57,9 +57,13 @@
 
 _US_WEST2_DOMAIN = "songbird-20201223060057342600000001.s3.us-west-2.amazonaws.com"
 _S3_PUBLIC_DOMAIN = os.environ.get("_WHYLABS_PRIVATE_S3_DOMAIN") or _US_WEST2_DOMAIN
+_WHYLABS_SKIP_CONFIG_READ = os.environ.get("_WHYLABS_SKIP_CONFIG_READ") or False
 
 
 def _check_whylabs_condition_count_uncompound() -> bool:
+    global _WHYLABS_SKIP_CONFIG_READ
+    if _WHYLABS_SKIP_CONFIG_READ:
```

Comment:
if I set this env var with `export _WHYLABS_SKIP_CONFIG_READ=False`, it will be parsed as a string and return True anyways.